go.mod

```diff
@@ -9,6 +9,8 @@ require (
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cobra v0.0.5
 	github.com/stretchr/testify v1.4.0
+	golang.org/x/lint v0.0.0-20200130185559-910be7a94367 // indirect
+	golang.org/x/tools v0.0.0-20200301222351-066e0c02454c // indirect
```

Comment:
Do we need these?
I think using `go mod tidy` make these unnecessary.

Reply:
Thanks, I tryed it and these packages doesn't need.